firmware/application/apps/pocsag_app.cpp

```diff
@@ -79,18 +79,21 @@ POCSAGAppView::POCSAGAppView(NavigationView& nav) {
         update_freq(f);
     };
 
-    // load app settings
+    // load app settings TODO: needed?
```

Comment:
It's used when people activate per app settings save and load.
I do not particularly like that option but some are using it.

Reply:
This PR started out as a "cleanup pocsag" fix and then scope creep :)